TuneUp/manifests/pkg.json

```diff
@@ -2,7 +2,7 @@
   "license": "",
   "file_hash": null,
   "name": "TuneUp",
-  "version": "1.0.11",
+  "version": "1.0.12",
```

Comment:
This is already changed in the other PR. Merged those changes. Can we remove this change from here

Reply:
done